perk-store-webapps/src/main/webapp/css/main.less

```diff
@@ -888,7 +889,12 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
   .buyButton{
     margin-top: 10px;
   }
-
+  .disabledBuyButton{
+    background-color: #E0E0E0 !important;
```

Comment:
Could you use a variable here  ?

Reply:
could not find it in variables list ! add new one?

Reply:
This color should havz been used in the product for disabled buttons, if not found we add a new one.

Reply:
it is used several times without variable. I'll add a new one.